### PR TITLE
809 fe sso create the initial structure of sso front

### LINKED ISSUE #809: [FE][SSO] Create the initial structure of `sso_front`, empty.

### PR DESCRIPTION
**Pull request by marmorrei**
Closes #809

### CODE REVIEW

File: sso_front/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="./src/assets/icons/favicon.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <title>ITA Wiki</title>
```
Comment (missvader) [inline sso_front/index.html:14]: Change title to ITA-SSO for now

File: sso_front/src/context/.gitkeep
Comment (missvader) [inline sso_front/src/context/.gitkeep:1]: Remove folder

File: sso_front/src/locales/.gitkeep
Comment (missvader) [inline sso_front/src/locales/.gitkeep:1]: Remove folder

File: config/dev/ui.yml
Comment (missvader) [inline config/dev/ui.yml:1]: This changes are from UI, remove them

**Review by missvader — CHANGES_REQUESTED** [review]
Some folders better not include them in initial structure. We have to consider that it's a different app from ITA-wiki, and it's possible that they may not be needed.

File: config/dev/ui.yml
Comment (marmorrei) [reply config/dev/ui.yml:1]: Ok, I am not sure how this ended up here! I will check.

**Review by missvader — APPROVED** [review]

**Pull request merged**